.github/workflows/ci-build.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     - uses: actions/checkout@v2
 
     - name: Install PHP
-      uses: shivammathur/setup-php@2.2.2
+      uses: shivammathur/setup-php@2.9.0
```

Comment:
FTR, ever since I've been using setup-php it's been extremely stable and the author is very diligent. Aka never have I experienced issues by just using `setup-php@2` and benefit from minor updates on the fly.

Reply:
Yup but 💩  happens, I guess it's better to lock to minor at least to have reproducible builds.